src/prefect/cli/_prompts.py

```diff
@@ -376,14 +351,6 @@ def prompt_schedules(console) -> List[DeploymentScheduleCreate]:
                 "active": is_schedule_active,
             }
 
-            if PREFECT_EXPERIMENTAL_ENABLE_SCHEDULE_CONCURRENCY:
```

Comment:
Can we remove this setting too?

Reply:
good call! removed in [72f1e9c](https://github.com/PrefectHQ/prefect/pull/15810/commits/72f1e9c23b0b42234de99215326396ac4e35167c)